
{% extends "base.html" %}

{% block title %} RNA sequence input form {% endblock %}

{% block menu %}
    <div id="menu">
      <ul>
      <li><a href="/rnavis">Home</a></li>
      <li><a href="/rnavis/new">Visualize</a></li>
      <li> <a href="/rnavis/list">List</a></li>
      <li class="current_page_item"><a href="/rnavis/references">References</a></li>
      <li><a href="/rnavis/contact">Contact</a></li>
      </ul>
     </div>
 {% endblock %}


{% block post %}
  <h2> References  </h2>
  </br>
   <p>1) Michiaki Hamada, Hisanori Kiryu, Kengo Sato, Toutai Mituyama, Kiyoshi  Asa (2009)    "Predictions of RNA secondary structure using generalized centroid estimators". Bioinformatics 25 (4): </br></br>


2) Michiaki Hamada, Hisanori Kiryu, Kengo Sato, Toutai Mituyama, Kiyoshi Asai (2009). "Predictions of RNA secondary structure by combining homologous sequence information". Bioinformatics 25 (12): i330.</br></br>

 

3)Do CB, Woods DA, Batzoglou S (2006). "CONTRAfold: RNA secondary structure prediction without physics-based models". Bioinformatics 22 (14): e90–8. 
</br></br>
 

4)Bindewald E, Kluth T, Shapiro BA (2010). "CyloFold: secondary structure prediction including   pseudoknots” Nucleic Acids Research Suppl (W): 368–72.
</br></br>
 

5)Richards F. M., Kundrot C. E. (1988). "Identification of structural motifs from protein coordinate data: secondary structure and first-level supersecondary structure". Proteins 3 (2): 71–84. doi:  10.1002/prot.340030202 PMID 3399495.
</br></br>
 

6)Kabsch W., Sander C. (1983). "Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features". Biopolymers 22 (12): 2577–2637. doi:10.1002/bip.360221211.PMID 6667333.[1]

 </br></br>

7)Frishman D., Argos P. (1995). "Knowledge-based protein secondary structure assignment". Proteins 23 (4): 566–579. doi:10.1002/prot.340230412. PMID 8749853.
</br></br>
{% endblock %}
